src/app/components/add-class-form/add-class-form.component.ts

```diff
@@ -47,7 +49,7 @@ export class AddClassFormComponent implements OnInit {
   toggleForm = () => this.formIsOpen = !this.formIsOpen;
 
   getTeachersToSelectData = () => {
-    this.teacherService.getTeachersToSelect().subscribe(data => this.teachers = data)
+    this.teacherService.getClassTeachersToSelect().subscribe(data => this.teachers = data)
   }
```

Comment:
Why class teachers?

Reply:
> Why class teachers?

Maybe  , when you create class , you must select a Class Teacher?
But you get all teachers but not a class teachers)))))